README.md

```diff
@@ -173,20 +210,19 @@ Or you can set them in the `.env` file.
 
 ## 💀 Continuous Mode ⚠️
 Run the AI **without** user authorisation, 100% automated.
-Continuous mode is not recommended. 
-It is potentially dangerous and may cause your AI to run forever or carry out actions you would not usually authorise. 
+Continuous mode is not recommended.
```

Comment:
Suggest "Continuous mode is not recommended unless you know what you're doing, since it can ramp up API costs or behave in unknown ways."